### PR TITLE
fix: duplicated entry

### PR DESCRIPTION
**Pull request by PassionPenguin**
SLD is the duplicated one for Second-level_Domain entry.

To keep consistency with TLD (top-level_domain, `Glossary/TLD`), removed `Glossary/second-level_domain` with redirects and moved its content to `Glossary/SLD`

### CODE REVIEW

File: files/en-us/glossary/sld/index.md [hunk 1]
```diff
@@ -6,6 +6,19 @@ page-type: glossary-definition
 
 {{GlossarySidebar}}
 
-An SLD ([Second Level Domain](/en-US/docs/Glossary/Second-level_Domain)) is the part of the domain name that is located right before a _Top Level Domain_ ({{Glossary("TLD")}}). For example, in `mozilla.org` the SLD is `mozilla` and the TLD is `org`.
+A Second Level Domain (SLD) is the part of the domain name that is located right before a Top Level Domain ({{Glossary("TLD")}}).
```
Comment (estelle) [inline files/en-us/glossary/sld/index.md:9]:
```suggestion
The **Second Level Domain (SLD)** is the part of the domain name that is located right before a {{Glossary("TLD", "Top Level Domain (TLD)")}}_.
```

File: files/en-us/glossary/sld/index.md [hunk 2]
```diff
@@ -6,6 +6,19 @@ page-type: glossary-definition
...
 
-See [Second Level Domain](/en-US/docs/Glossary/Second-level_Domain) for more information.
+For example, in `mozilla.org` the SLD is `mozilla` and the TLD is `org`.
+
+A domain name is not limited to a TLD and an SLD. Additional subdomains can be created in order to provide additional information about various functions of a server or to delimit areas under the same domain. For example, `www` is a commonly used subdomain to indicate the domain points to a web server.
```
Comment (estelle) [inline files/en-us/glossary/sld/index.md:13]:
```suggestion
A domain name is not limited to a TLD and SLD. Additional subdomains can be created to provide additional information about various server functions or to delimit areas under the same domain. For example, `www` is a commonly used subdomain to indicate that the domain points to a web server.
```

**Review by estelle — APPROVED** [review]
LGTM, but made some bolding a grammar suggestions for your consideration.

**Comment by PassionPenguin** [comment]
> LGTM, but made some bolding a grammar suggestions for your consideration. 

Thanks for your suggestions :)

**Pull request merged**